pkg/resources/custom_diffs.go

```diff
@@ -22,6 +22,32 @@ func NestedIntValueAccountObjectComputedIf(key string, parameter sdk.AccountPara
 	)
 }
 
+// NestedStringValueAccountObjectComputedIf is NestedValueComputedIf,
+// but dedicated for account level objects with string properties.
+func NestedStringValueAccountObjectComputedIf(key string, parameter sdk.AccountParameter) schema.CustomizeDiffFunc {
+	return NestedValueComputedIf(
+		key,
+		func(client *sdk.Client) (*sdk.Parameter, error) {
+			return client.Parameters.ShowAccountParameter(context.Background(), parameter)
+		},
+		func(v any) string { return v.(string) },
+	)
+}
+
+// NestedBoolValueAccountObjectComputedIf is NestedValueComputedIf,
+// but dedicated for account level objects with bool properties.
+func NestedBoolValueAccountObjectComputedIf(key string, parameter sdk.AccountParameter) schema.CustomizeDiffFunc {
+	return NestedValueComputedIf(
+		key,
+		func(client *sdk.Client) (*sdk.Parameter, error) {
```

Comment:
I what sense? You mean I could extract another function where account param function could be used across those 3 functions? Plus, I believe I'll be able to use plain values instead of nested ones, so I could remove those and adjust the new ones.